esmvaltool/utils/cmorizers/obs/cmorize_obs.py

```diff
@@ -184,6 +184,9 @@ def execute_cmorize():
         timestamp1.strftime(timestamp_format))
 
     logger.info(70 * "-")
+    # check if the inputdir actually exists
+    if not os.path.isdir(config_user["rootpath"]["RAWOBS"][0]):
+        logger.error("input_dir does not exist")
```

Comment:
Maybe better:
```
logger.error("input_dir_path %s does not exist", config_user["rootpath"]["RAWOBS"][0])
```
?

Reply:
Thanks, yes, I implemented as you suggest now.